features/all-positions/AllLiquidations.tsx

```diff
@@ -0,0 +1,330 @@
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  Tooltip,
+} from "@material-ui/core";
+import styled from "styled-components";
+import { utils } from "ethers";
+import { useState, useEffect } from "react";
+
+import EmpState from "../../containers/EmpState";
+import Collateral from "../../containers/Collateral";
+import Token from "../../containers/Token";
+import EmpSponsors from "../../containers/EmpSponsors";
+import Etherscan from "../../containers/Etherscan";
+
+import { isPricefeedInvertedFromTokenSymbol } from "../../utils/getOffchainPrice";
+
+interface SortableTableHeaderProps {
+  children: React.ReactNode;
+  sortField: number;
+}
+
+const Link = styled.a`
+  color: white;
+  font-size: 18px;
+`;
+
+const ClickableText = styled.span`
+  &:hover {
+    cursor: pointer;
+    opacity: 0.6;
+  }
+  text-align: end;
+  user-select: none;
+`;
+
+const PageButtons = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  margin-top: 2em;
+  margin-bottom: 2em;
+`;
+
+type FadedDiv = {
+  faded: boolean;
+};
+
+const Arrow = styled.div<FadedDiv>`
+  color: ${({ theme }) => theme.primary1};
+  opacity: ${(props) => (props.faded ? 0.3 : 1)};
+  padding: 0 20px;
+  user-select: none;
+  :hover {
+    cursor: pointer;
+  }
+`;
+
+enum SORT_FIELD {
+  MAX_DISPUTABLE_PRICE,
+  LOCKED_COLLATERAL,
+  TOKENS_LIQUIDATED,
+  LIQUIDATION_TIMESTAMP,
+}
+
+const FIELD_TO_VALUE: { [sortField: number]: string } = {
+  [SORT_FIELD.MAX_DISPUTABLE_PRICE]: "maxDisputablePrice",
+  [SORT_FIELD.LOCKED_COLLATERAL]: "lockedCollateral",
+  [SORT_FIELD.TOKENS_LIQUIDATED]: "tokensLiquidated",
+  [SORT_FIELD.LIQUIDATION_TIMESTAMP]: "liquidationTimestamp",
+};
+
+const ITEMS_PER_PAGE = 10;
+
+const AllLiquidations = () => {
+  const { empState } = EmpState.useContainer();
+  const { priceIdentifier: priceId, liquidationLiveness } = empState;
+  const { symbol: tokenSymbol } = Token.useContainer();
+  const { symbol: collSymbol } = Collateral.useContainer();
+  const { liquidations } = EmpSponsors.useContainer();
+  const { getEtherscanUrl } = Etherscan.useContainer();
+
+  // Pagination
+  const [page, setPage] = useState<number>(1);
+  const [maxPage, setMaxPage] = useState<number>(1);
+
+  // Sorting
+  const [sortDirection, setSortDirection] = useState<boolean>(true);
+  const [sortedColumn, setSortedColumn] = useState<number>(
+    SORT_FIELD.TOKENS_LIQUIDATED
+  );
+
+  // Set max page depending on # of liqs
+  useEffect(() => {
+    setMaxPage(1);
+    setPage(1);
+
+    if (liquidations) {
+      let extraPages = 1;
+      if (Object.keys(liquidations).length % ITEMS_PER_PAGE === 0) {
+        extraPages = 0;
+      }
+      setMaxPage(
+        Math.floor(Object.keys(liquidations).length / ITEMS_PER_PAGE) +
+          extraPages
+      );
+      // This will set maxPage to 0 if there are no liqs.
```

Comment:
nit: won't it set it to 1 (what it's set to at the top of this function before the if)?

Reply:
Yeah agreed I removed this comment